client/src/components/_common/Section/SectionMessages.css

```diff
@@ -0,0 +1,10 @@
+/* Increase minimum height for at least one message */
+/* FAQ: Edit global style based on local state side effect */
+/* TODO: Update Dashboard.(s)css, DataFiles.css to use this */
```

Comment:
This will be done in the next FP-385 PR (where all sections use these new components).